internal/cli/serverless/auditlog/config/update.go

```diff
@@ -0,0 +1,537 @@
+  $ %[1]s serverless audit-log config update -c <cluster-id> --unredacted
+
+  Enable database audit logging in non-interactive mode:
+  $ %[1]s serverless audit-log config update -c <cluster-id> --enabled
+
+  Disable database audit logging in non-interactive mode:
+  $ %[1]s serverless audit-log config update -c <cluster-id> --enabled=false`, config.CliName),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.MarkInteractive(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			d, err := h.Client()
+			if err != nil {
+				return err
+			}
+			ctx := cmd.Context()
+
+			var clusterID string
+			var unredacted, enabled *bool
+			var rotationIntervalMinutes, rotationSizeMib int32
+			var cloudStorage auditlog.CloudStorageTypeEnum
+			// s3
+			var s3URI, accessKeyID, secretAccessKey, s3RoleArn string
+			// gcs
+			var gcsURI, gcsServiceAccountKey string
+			// azure
+			var azBlobURI, azBlobSasToken string
+			// oss
+			var ossURI, ossAccessKeyID, ossAccessKeySecret string
+			if opts.interactive {
+				if !h.IOStreams.CanPrompt {
+					return errors.New("The terminal doesn't support interactive mode, please use non-interactive mode")
+				}
+				project, err := cloud.GetSelectedProject(ctx, h.QueryPageSize, d)
+				if err != nil {
+					return err
+				}
+				selectedCluster, err := cloud.GetSelectedCluster(ctx, project.ID, h.QueryPageSize, d)
+				if err != nil {
+					return err
+				}
+				clusterID = selectedCluster.ID
+
+				fieldName, err := cloud.GetSelectedField(mutableFields, "Choose one field to config:")
+				if err != nil {
+					return err
+				}
+
+				switch fieldName {
+				case string(Unredacted):
+					prompt := &survey.Confirm{
+						Message: "unredact the database audit log?",
+						Default: false,
+					}
+					var confirm bool
+					err = survey.AskOne(prompt, &confirm)
+					if err != nil {
+						if err == terminal.InterruptErr {
+							return util.InterruptError
+						} else {
+							return err
+						}
+					}
+					if confirm {
+						unredacted = aws.Bool(true)
+					} else {
+						unredacted = aws.Bool(false)
+					}
+				case string(Enabled):
+					prompt := &survey.Confirm{
+						Message: "enable database audit logging?",
+						Default: false,
+					}
+					var confirm bool
+					err = survey.AskOne(prompt, &confirm)
+					if err != nil {
+						if err == terminal.InterruptErr {
+							return util.InterruptError
+						} else {
+							return err
+						}
+					}
+					if confirm {
+						enabled = aws.Bool(true)
+					} else {
+						enabled = aws.Bool(false)
+					}
+				case string(CloudStorage):
+					cloudStorage, err = GetSelectedCloudStorage()
+					if err != nil {
+						return err
+					}
+					selectedAuthType, err := GetSelectedAuthType(cloudStorage, *selectedCluster.CloudProvider)
+					if err != nil {
+						return err
+					}
+					switch selectedAuthType {
+					// Both S3 and OSS supports ACCESS_KEY
```

Comment:
isn't OSS use `OSSCLOUDSTORAGEOSSAUTHTYPE_ACCESS_KEY`?

Reply:
I use a string to judge auth type.
Both OSSCLOUDSTORAGEOSSAUTHTYPE_ACCESS_KEY and S3 ACCESS_KEY are `ACCESS_KEY` in string type